### DISCUSSION TITLE
ERROR - Review_id in Featured-reviews.csv

### DISCUSSION THREAD

**Discussion by Jer0** [omkarcloud/google-maps-scraper#105] [General]
Hello again

After a while exporting data and seeing what was being exported, I came across an error in the file generated "featured-reviews-of...csv, the review_id column does not have the correct value

I have changed the element in the "get_user_reviews" function in extract_data.py

review_id = element[10]

After looking at the two csvs (detailed_reviews and featured_reviews), they are identical except for the column, review_photos, which exists in featured but does not exist in detailed, why not? Wouldn't it be easier to have everything in the same excel and enter a column that says if that review is featured or not?

For the rest, the script performs great!

**Comment by Chetan11-dev** [comment]
Thank you, @Jer0, for initiating the discussion and utilizing our Google Maps Scraper Tool. Your solution is greatly appreciated, and in response, I have updated the Scraper. You can download the latest version from the Pro Link. Additionally, I'd like to clarify that 'featured-reviews' refer to the select few reviews (typically a maximum of 8) showcased on a Business Page. In contrast, 'detailed-reviews' encompass all the reviews of a place, potentially ranging from hundreds to thousands, which are scraped from the business listings.
